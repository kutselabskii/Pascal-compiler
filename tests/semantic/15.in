program abc;

const a = 2 / 0;
begin
end.