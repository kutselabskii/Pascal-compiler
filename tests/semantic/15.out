Line 3 : Col 16 : Semantic error :: Division by zero
